src/modules/previewpane/common/Telemetry/TelemetryBase.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Eventing.Reader;
+using System.Diagnostics.Tracing;
+
+namespace PreviewHandlerCommon.Telemetry

```

Comment:
@udit3333   Is there a task to verify that we get the expected telemetry events once this is built out of an official build?  @enricogior  / @crutkas :  We may just want to have one issues as part of our release checklist if there isn't which included validating the telemetry for each of the powertoys.   I'm not sure how to do this.  Also it would be great if we could validate this prior to getting in an official build.  Does the 'mock' telemetry nuget fire log events anywhere?

Reply:
> @udit3333 Is there a task to verify that we get the expected telemetry events once this is built out of an official build? 

@ryanbodrug-microsoft Currently there is no task for tracking this I will add a task for this

 > Also it would be great if we could validate this prior to getting in an official build. Does the 'mock' telemetry nuget fire log events anywhere?

We did validate the telemetry events fired with Windows Performance Recorder for both `Svg` and `Markdown` preview handler. I just noticed the `.wprp` is being replaced by a binary file[ in PR change](https://github.com/microsoft/PowerToys/commit/1a01818fb6dab08c98419b53834decf99d527fad). I will fix the file back to xml file and validate the events again before merging the PR.

Reply:
> I will fix the file back to xml file and validate the events again before merging the PR.
 
Re added the `.wprp` file. Validated the telemetry events are being fired locally on dev machine.
![image](https://user-images.githubusercontent.com/15176579/76467375-bb850780-63a6-11ea-90f5-6beea31c88cb.png)